CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## 0.51.0 / 2021-09-22
+
+* [FEATURE] Add `metricRelabelings` field to the Probe CRD for configuring the metric relabel configs. #4226
```

Comment:
Should this be like `Added` instead of `Add` ?

Reply:
yes I agree!